projectroles/plugins.py

```diff
@@ -608,6 +620,17 @@ def get_extra_data_link(self, _extra_data, _name):
         # TODO: Implement this in your app plugin
         return None
 
+    def validate_form_app_settings(self, app_settings, user=None):
```

Comment:
Only project and site apps are expected to provide app settings, so for `BackendPluginPoint` this is not needed.

Reply:
But this `validate_form_app_settings` is for `SiteAppPluginPoint`. The `BackendPluginPoint` has no such method